ms-authentication/routes/controller.go

```diff
@@ -0,0 +1,34 @@
+// Copyright © 2022 Intel Corporation. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+package routes
+
+import (
+	"fmt"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/interfaces"
+)
+
+type Controller struct {
+	service interfaces.ApplicationService
+}
+
+func NewController(service interfaces.ApplicationService) Controller {
+	return Controller{
+		service: service,
```

Comment:
Common practice is to also set the logger so each logging call doesn't have to go thru the service API
```suggestion
		service: service,
		lc: service.LoggerClient(),
```

Reply:
@jim-wang-intel I think has a conflicting opinion any thoughts?

Reply:
i am fine either way, with lenny's suggestion, just another copy of logging client here.

Reply:
This makes the code using the logger cleaner and easier to read. Commonly used approach in the EdgeX code.